exporter/collector/config.go

```diff
@@ -28,23 +28,33 @@ const (
 type Config struct {
 	ProjectID string `mapstructure:"project"`
 	UserAgent string `mapstructure:"user_agent"`
-	Endpoint  string `mapstructure:"endpoint"`
```

Comment:
Saw your reply on the older PR. IIUC, if we are using the new exporter it will throw an error if this option is supplied. If we are using the old one, it will use the old config struct so should be fine. 

Is that right?

Reply:
Yes